stixcore/idb/tests/test_manager.py

```diff
@@ -1,16 +1,22 @@
 import os
+from datetime import datetime
 from pathlib import Path
 
 import pytest
 
 from stixcore.idb.manager import IdbManager
 from stixcore.idb.idb import IDB
+import shutil
 
 
 @pytest.fixture
 def idbManager():
     return IdbManager(Path(os.path.abspath(__file__)).parent / 'data')
 
+def teardown_function():
```

Comment:
I think this is a bit of a code smell should probably use pytest tmpdir and the download should be to a configurable location?

Reply:
open an issue here: https://github.com/i4Ds/STIXCore/issues/16